### PR TITLE
Update Send-Many-Interfaces

### PR DESCRIPTION
**Pull request by OmurCataltepe**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,18334 @@
+{
+  "name": "stacks-send-many",
```
Comment (friedger) [inline package-lock.json:2]: we use just yarn, package-lock.json should be removed.

File: package.json [hunk 2]
```diff
@@ -16,13 +16,16 @@
...
+    "@types/react-dom": "^17.0.3",
     "bn.js": "^5.2.0",
     "c32check": "^1.1.2",
     "jotai": "^0.16.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-input-switch": "^2.2.2",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "yarn": "^1.22.10"
```
Comment (friedger) [inline package.json:28]: Do we need that?

File: src/App.js [hunk 3]
```diff
@@ -74,33 +74,27 @@ function Content({ userSession }) {
...
-          />
-          <SendManyDetails
-            path="/txid/:txId"
-            decentralizedID={decentralizedID}
-            userSession={userSession}
-          />
-          {!authenticated && <Landing path="/" />}
-          {decentralizedID && (
-            <>
+      {!authenticated && <Landing />}
```
Comment (friedger) [inline src/App.js:77]: Changes in this file should be reverted

File: src/components/SendManyInput.js [hunk 4]
```diff
@@ -1,38 +1,98 @@
...
-      to: to.current.value.trim(),
-      stx: stxValue.current.value.trim(),
-      memo: memo.current.value.trim(),
-    };
-    return value;
-  };
+import React, {useState} from 'react';
 
+ 
+export function SendManyInput({ onKeyUp, key }) {
```
Comment (friedger) [inline src/components/SendManyInput.js:4]: This is a component that represents just one row.

File: src/styles/style.css [hunk 5]
```diff
@@ -55,3 +55,638 @@ inspect {
...
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-items: center;
+	padding: 15px;
+	background: #e6e6e6
+}
+
+.wrap-contact100 {
+	width: 920px;
```
Comment (friedger) [inline src/styles/style.css:76]: Why 920px ?

File: src/styles/style.css [hunk 6]
```diff
@@ -55,3 +55,638 @@ inspect {
 .monster {
   filter: grayscale(100);
 }
+
+
+.container-contact100 {
```
Comment (friedger) [inline src/styles/style.css:60]: Do we need all these styles? Is bootstrap not enough?

**Review by friedger — CHANGES_REQUESTED** [review]

**Comment by OmurCataltepe** [comment]
Probably I worked on an old version. I think it's better to get the last version and just move the SendManyInput.js and style.css. Because I only changed these files, I don't have any idea about the other files.

**Pull request closed without merging**